metadata.json

```diff
@@ -25,6 +25,7 @@
     { "operatingsystem": "Scientific", "operatingsystemrelease": [ "6", "7" ] },
     { "operatingsystem": "Fedora",     "operatingsystemrelease": [ "19" ] },
     { "operatingsystem": "Debian",     "operatingsystemrelease": [ "6", "7" ] },
-    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] }
+    { "operatingsystem": "Ubuntu",     "operatingsystemrelease": [ "12.04", "14.04" ] },
+    { "operatingsystem": "Windows",    "operatingsystemrelease": [ "7", "2012 R2" ] }
```

Comment:
Let me take a look at this. I saw more details in the Puppet Labs docs I used in my research.